helm/cluster-azure/ci/test-wc-restricted-access-values.yaml

```diff
@@ -0,0 +1,13 @@
+metadata:
+  name: test-wc-restricted
+  organization: test
+  servicePriority: lowest
+
+connectivity:
+  allowedCIDRs:
+    - "1.1.1.1"
```

Comment:
should we add the netmask here to make it more clear ? `/32` 

also probably to the validation of the schema object

Reply:
i added a `replace "/" "_"`